src/main/java/processor/DataProcessorWithStrategy.java

```diff
@@ -0,0 +1,29 @@
+package processor;
+
+import java.util.List;
+import model.Operation;
+import processor.strategy.ProcessingStrategy;
+import processor.strategy.ProcessingStrategyImpl;
+import service.DataStorageService;
+import service.impl.DataStorageServiceImpl;
+import storage.OperationalStorage;
+
+public class DataProcessorWithStrategy implements DataProcessor {
+    private static final String DATA_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int VALUE_INDEX = 2;
+    private final DataStorageService storageService = new DataStorageServiceImpl();
```

Comment:
Common Mistakes: "All services should be independent. We shouldn’t have Strategy and call its methods in CsvFileReaderService, or we shouldn’t have CsvFileWriterService and call its methods in the Strategy class."
I think you need Dao here.